src/components/DashboardPage/ProfileDashboardData.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { PROFILE_ACTIONS } from '../../redux/actions/profileActions';
+
+const mapStateToProps = reduxState => ({
+    profile: reduxState.profile
```

Comment:
there's nothing in the redux state called 'profile'. Check the store, and remember you can only use those.